make ZLIB_DEFLATE visible, so that we can choose whether we want it built-in
or as a module

--- a/lib/Kconfig
+++ b/lib/Kconfig
@@ -98,7 +98,7 @@ config ZLIB_INFLATE
 	tristate
 
 config ZLIB_DEFLATE
-	tristate
+	tristate "Zlib compression"
 
 config LZO_COMPRESS
 	tristate
